### ISSUE TITLE
Item state resetting to 0 when using `datastore` database

### ISSUE THREAD

**Issue by negrifelipe** [EvolutionPlugins/Kits#7]
For some reason when you add guns or any item that has a state to the kit, the state of it only works once after that use the item starts to be received  without attachments or any state at all.
Like here:
![image](https://github.com/EvolutionPlugins/Kits/assets/64453816/2e642806-a14f-485b-9d0e-f184c3e28f8b)

**Comment by DiFFoZ** [comment]
Are you using `datastore` database? Someone already reported about that in my [discord](https://discord.com/channels/764502843906064434/764505338267303966/1040423716716298261).

Still, I don't know how to fix it and why it is happening. Maybe it is a bug from YAML lib or from OpenMod.
**Easy workaround is just using MySQL database**

**Comment by negrifelipe** [comment]
Yes, I am using data store. Will change to MySql soon. Thanks apart from that the plugin works awesome!

**Comment by negrifelipe** [comment]
Using MySQL seems to have fixed it. Maybe switching to json instead of openmod's data store would be a quick solution. 
The yaml lib seems to work but not all the time that's why sometimes kits works as expected. For example if i reload openmod and the plugin loads again the first that uses a kit gets it right

**Comment by DiFFoZ** [comment]
Suggestion to fix: try to copy the state of the item when adding to the store
